src/site/content/en/secure/what-is-mixed-content/index.md

```diff
@@ -0,0 +1,157 @@
+</figure>
+
+### Active mixed content
+
+Active mixed content poses a greater threat than passive mixed content.
+An attacker can intercept and rewrite active content,
+thereby taking full control of your page or even your entire website.
+This allows the attacker to change anything about the page,
+including displaying entirely different content,
+stealing user passwords or other login credentials,
+stealing user session cookies,
+or redirecting the user to a different site entirely.
+
+Due to the severity of this threat,
+many browsers already block this type of content by default to protect users,
+but functionality varies between browser vendors and versions.
+
+The following contains examples of active mixed content:
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/active-mixed-content?previewSize=100"
+    title="active-mixed-content on Glitch"
+    allow="encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+[Load the example over HTTP](http://active-mixed-content.glitch.me/) and you will see the content that has been blocked.
+
+Blocked content will be detailed in the Chrome DevTools console.
+
+<figure class="w-figure">
+  <img class="w-screenshot"
+      src="active-mixed-content.jpg"
+      alt="Chrome DevTools showing the warnings displayed when active mixed content is blocked">
+</figure>
+
+## The mixed content specification
+
+Browsers follow the [mixed content specification](https://w3c.github.io/webappsec-mixed-content/),
+which defines the [**optionally blockable content**](https://w3c.github.io/webappsec-mixed-content/#optionally-blockable-mixed-content)
+and [**blockable content**](https://w3c.github.io/webappsec-mixed-content/#category-blockable) categories.
+
+From the spec,
+a resource qualifies as optionally blockable content
+"when the risk of allowing its usage as mixed content is outweighed by the risk of breaking significant portions of the web";
+this is a subset of the passive mixed content category described above.
+
+All content that is not **optionally blockable** is considered **blockable**,
+and should be blocked by the browser.
+
+In recent years, [HTTPS usage has risen dramatically](https://transparencyreport.google.com/https/overview),
+and has become the clear default on the web.
+This makes it more feasible now for browsers to consider blocking all mixed content,
+even those subresource types defined in the [mixed content specification](https://w3c.github.io/webappsec/specs/mixedcontent/) as **optionally blockable**.
+This is why we now see Chrome taking a stricter approach to these subresources.
+
+### Older browsers
+
+It is important to remember that not every visitor to your website uses the most up-to-date browsers.
+Different versions from different browser vendors each behave differently with mixed content.
```

Comment:
```suggestion
Different versions from different browser vendors each treat mixed content differently.
```
Just a nitpick-y rephrase because I feel that this makes it clearer that the browser is actually choosing what to do with mixed content